entrypoint.sh

```diff
@@ -18,7 +18,8 @@ publish_dependencies_as_layer(){
 
 publish_function_code(){
 	echo "Deploying the code itself..."
-	zip -r code.zip . -x \*.git\*
+	zip -r code.zip . -x "${SOURCE_DIR}"
```

Comment:
The `-x` flag excludes the following argument (which is why it was used to exclude the hidden `.git` directory). This would mean that the `SOURCE_DIR` would be excluded from zipping. Is that the intended behaviour? Also: how is this variable set? Would there be use cases where it isn't? Could you provide documentation for this/edit the readme?